aten/src/ATen/core/TransformationHelper.h

```diff
@@ -67,7 +67,7 @@ C10_HOST_DEVICE inline T uniform_int_transformation(V val) {
 template <typename T, typename V>
 C10_HOST_DEVICE inline dist_acctype<T> uniform_real_transformation(V val, T from, T to) {
   constexpr auto MASK = static_cast<V>((static_cast<uint64_t>(1) << std::numeric_limits<T>::digits) - 1);
-  constexpr auto DIVISOR = static_cast<T>(1) / (static_cast<uint64_t>(1) << std::numeric_limits<T>::digits);
+  constexpr auto DIVISOR = static_cast<dist_acctype<T>>(1) / (static_cast<uint64_t>(1) << std::numeric_limits<T>::digits);
```

Comment:
@pbelevich please let me know if this makes sense to you. (c10::Half does not have contexpr divide operator, but on the other half, since we cast it to `dist_acctype<T>` on the next line, it does not have to be of type `T`

Reply:
agree, makes sense